test/e2e/alphaupdate/update_test.go

```diff
@@ -295,3 +354,48 @@ func validateCustomCodePreservation(projectDir string) {
 	Expect(err).NotTo(HaveOccurred())
 	Expect(string(content)).To(ContainSubstring(controllerImplementation))
 }
+
+func validateConflictMarkers(projectDir string, expectMarkers bool) {
```

Comment:
I think it is a little overwhelming for what we need.
but lets move forward and after we improve in a follow up